### PR TITLE
add bitmap generation support in zap for python cluster objects

### PR DESCRIPTION
**Pull request by yunhanw-google**
Previously we don't have bitmap generation support for python cluster objects. In this PR, we add this support

### CODE REVIEW

**Review by yufengwangca — APPROVED** [review]

**Review by jmeg-sfy — APPROVED** [review]

File: src/controller/python/templates/python-cluster-Objects-py.zapt [hunk 1]
```diff
@@ -56,6 +56,17 @@ class {{asUpperCamelCase name}}(Cluster):
 
 {{/zcl_enums}}
 
+{{#zcl_bitmaps}}
+{{#first}}
+    class Enums:
+{{/first}}
+        class {{asType label}}(IntEnum):
```
Comment (tehampson) [inline src/controller/python/templates/python-cluster-Objects-py.zapt:63]: Curious, why are you not using `IntFlag`? Would you not want a value of this type to be a combination of these flags?

File: src/controller/python/templates/python-cluster-Objects-py.zapt [hunk 2]
```diff
@@ -56,6 +56,17 @@ class {{asUpperCamelCase name}}(Cluster):
 
 {{/zcl_enums}}
 
+{{#zcl_bitmaps}}
+{{#first}}
+    class Enums:
```
Comment (bzbarsky-apple) [inline src/controller/python/templates/python-cluster-Objects-py.zapt:61]: Is this really supposed to be Enums, not Bitmaps?

File: src/controller/python/templates/python-cluster-Objects-py.zapt (hunk 1 above)
Comment (yunhanw-google) [reply src/controller/python/templates/python-cluster-Objects-py.zapt]: thanks, updated.

File: src/controller/python/templates/python-cluster-Objects-py.zapt (hunk 2 above)
Comment (yunhanw-google) [reply src/controller/python/templates/python-cluster-Objects-py.zapt]: thanks, updated

**Pull request merged**